cmd/guacone/cmd/oci.go

```diff
@@ -169,6 +264,47 @@ func validateOCIFlags(gqlEndpoint, headerFile, csubAddr string, csubTls, csubTls
 	return opts, nil
 }
 
+func validateOCIRegistryFlags(gqlEndpoint, headerFile, csubAddr string, csubTls, csubTlsSkipVerify bool,
+	queryVulnIngestion bool, queryLicenseIngestion bool, queryEOLIngestion bool, args []string) (ociRegistryOptions, error) {
+	var opts ociRegistryOptions
+	opts.graphqlEndpoint = gqlEndpoint
+	opts.headerFile = headerFile
+	opts.queryVulnOnIngestion = queryVulnIngestion
+	opts.queryLicenseOnIngestion = queryLicenseIngestion
+	opts.queryEOLOnIngestion = queryEOLIngestion
+
+	csubOpts, err := csub_client.ValidateCsubClientFlags(csubAddr, csubTls, csubTlsSkipVerify)
```

Comment:
I think csub is optional here as well? perhaps the same behavior with https://github.com/guacsec/guac/pull/2241/files#diff-3b3eb27190ed25c9e08b6ea2e170f66360c3df0f26fd382e71bb96a294386a37R225 would be ok?